tastypie/resources.py

```diff
@@ -61,11 +61,6 @@ def sanitize(text):
     return escape(text).replace('&#39;', "'").replace('&quot;', '"')
 
 
-class NOT_AVAILABLE:
```

Comment:
Added in 075b2c4ca012374b67f8263376c63756d86c2ada, the one place in the code that used this was removed.